internal/cli/cmd/version.go

```diff
@@ -15,23 +15,45 @@
 package cmd
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"net/http"
 	"runtime"
 	"strings"
 
-	"github.com/cilium/cilium-cli/defaults"
```

Comment:
Please keep the `github.com/cilium/cilium-cli` import in a separate import group before any other 3rd party imports, i.e.

```go
import (
	"context"
	"fmt"
	"io"
	"net/http"
	"runtime"
	"strings"
	
	"github.com/cilium/cilium-cli/defaults"
	"github.com/cilium/cilium-cli/internal/k8s"
	
	"github.com/spf13/cobra"
)
```